homeassistant/components/light/lightwave.py

```diff
@@ -0,0 +1,112 @@
+"""
+homeassistant.components.light.lightwave.
+
+Implements LightwaveRF lights.
+
+
+My understanding of the LightWave Hub is that devices cannot be discovered
+so must be registered manually. This is done in the configuration file:
+
+switch:
+  - platform: lightwave
+    devices:
+      R1D2:
+        name: Room one Device two
+      R2D1:
+        name: Room two Device one
+
+Each device requires an id and a name. THe id takes the from R#D#
+where R# is the room number and D# is the device number.
+
+For more details on the api see: https://api.lightwaverf.com/
+"""
```

Comment:
Please align the file header according the other platforms/components.

Reply:
I have added a link to https://home-assistant.io/components/light.lightwave/. But how do I actually create this documentation?